README.md

```diff
@@ -1,13 +1,19 @@
-Demo Express app for projects participating in Exchange Discover program
+A demo Express app for projects participating in xExchange Discover program.
 
 ## Quick start
 
-1. Run `npm install` in the project directory
-2. Run `npm start` to start the app on port 3000 (this can be changed in `server.js`)
+1. Run `npm install` to install the app.
+2. Run `npm run create-env-file` to generate a `.env` configuration file. 
+3. Update `WALLET_PATH`, `WALLET_PASSWORD` and `PROJECT_ID` inside the newly created `.env` file.
+4. Run `npm start` to start the app on port 3000 (this can be changed in `server.js`).
 
 ## Configuration
 
-The application expects a wallet (by default `signer.pem`) file to exist a the root level
+After running `npm run create-env-file`, you will need to updade the values inside.
 
-You can update the path for this inside `generateResponses.js`
+The default wallet path is `./signer_wallet.json`. Provide the file or ensure the file exists at the provided path before running the app.
+
+## Disclaimer
```

Comment:
Let's put the disclaimer before `## Quick start`.

And let's make the text even more warning:

```
This app is for illustration purposes only and is not suited for production environments as it lacks validation and sanitization of the endpoint parameters.
```